examples/claude.py

```diff
@@ -1,6 +1,7 @@
 import ell  # type: ignore
+import anthropic
 
-@ell.simple(model="claude-3-5-sonnet-20240620", max_tokens=100)
+@ell.simple(model="claude-3-5-sonnet-20241022", client=anthropic.Anthropic(), max_tokens=100)
```

Comment:
Do we need the client and anthropic import? 

ell should register anthropic as the default client for all anthropic models. 

It’s nice to show how this can also be explicit, really just wondering if this was needed for the code to run.

Reply:
yeah please remove the client=...